### PR TITLE
hu translation and 1-character ellipses

### PR DESCRIPTION
**Pull request by ovari**
Suggestion:
Change `Self Message` to `Message to Self`
Change `Self Messages` to `Messages to Self`

Self message could mean automated messages.

If you want this change please advise and we can send a pull request for L58 and L61.

Thank you

### CODE REVIEW

File: public/_locales/de/messages.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "message": "Wunderschön visualisierte Statistiken für Thunderbird E-Mail-Konten"
   },
   "options": {
+
```
Comment (devmount) [inline public/_locales/de/messages.json:6]: Please remove this additional line

**Review by devmount — CHANGES_REQUESTED** [review]
Awesome! Many thanks for these additions and corrections 👏🏻 so glad we now have the Hungary translation as well 👍🏻
There was just one unnecessary additional line break - as soon as you remove that, I'll merge this PR

**Comment by devmount** [comment]
> Self message could mean automated messages.

Yes, I wasn't so content about this wording either. Please [start a corresponding discussion thread](https://github.com/devmount/third-stats/discussions/new?category=changes) so that we can find the best name for this option together. Thank you very much!

File: public/_locales/de/messages.json (hunk 1 above)
Comment (ovari) [reply public/_locales/de/messages.json]: Done

**Review by devmount — APPROVED** [review]
Wow that was lightning fast 👏🏻 Thanks again!

**Comment by ovari** [comment]
@devmount please add any new strings in English to the hu/messages.json file as we edit the file using GitHub's editor, i.e. online, and please send a message. Thank you

**Comment by ovari** [comment]
Please let us know when your add-on with the hu locale will be available on [ATN](https://addons.thunderbird.net/thunderbird/addon/thirdstats/) so we can check the hu version. Thank you once again.

**Comment by ovari** [comment]
Does https://gitlocalize.com/repo/5505 automatically add new translations? Thank you

**Comment by devmount** [comment]
As soon as I've implemented the hu version, I'll add Hungary to our translation platform [Gitlocalize](https://gitlocalize.com/repo/5505). If you registered just let me know, I can then add you as translator and you will have a neat web interface showing you all missing translation keys in the future. You can create a review request which I can merge as a PR then.

**Comment by ovari** [comment]
> As soon as I've implemented the hu version, I'll add Hungary to our translation platform [Gitlocalize](https://gitlocalize.com/repo/5505). If you registered just let me know, I can then add you as translator and you will have a neat web interface showing you all missing translation keys in the future. You can create a review request which I can merge as a PR then.

There is a country call `Magyarország` which the English call `Hungary`. The language is `magyar` which the English call `Hungarian`.

Happy to just translate the strings in GitHub if you can just add the missing strings in English.

Thank you

**Comment by devmount** [comment]
> There is a country call `Magyarország` which the English call `Hungary`. The language is `magyar` which the English call `Hungarian`.

Of course, sorry, I'm not a native English speaker 😅.

> Happy to just translate the strings in GitHub if you can just add the missing strings in English.

I see, no problem, I'll find a way.

**Pull request merged**